src/language_server_protocol.rs

```diff
@@ -1759,6 +1774,74 @@ impl LanguageClient {
         Ok(())
     }
 
+    pub fn languageClient_handleCodeLensAction(&self, params: &Value) -> Fallible<Value> {
+        let filename = self.vim()?.get_filename(params)?;
+        let line = self.vim()?.get_position(params)?.line;
+
+        let mut code_lens: Vec<CodeLens> =
+            self.get(|state| state.code_lens.get(filename.as_str()).cloned().unwrap())?;
+        code_lens.retain(|cl| cl.range.start.line == line);
+        if code_lens.len() != 1 {
```

Comment:
What would properly be in that case? Maybe trigger the last action (that one should be the one that is showing if there are multiple codeLenses for a single line)? We could eventually refactor that to show a list of actions and let the user decide, but I don't know if that's worth it, because we don't have a way of displaying multiple codelenses for the same line now, and I don't think anyone is going to trigger an action for a codelens they can't see.